app/models/user.rb

```diff
@@ -287,7 +287,7 @@ def inactive_message
     I18n.t(:inactive_html, path_to_contact: '/contact').html_safe
   end
 
-  def avatar_url(size=nil,avatar_kind=nil)
+  def avatar_url(size=nil, kind=nil)
```

Comment:
Barg, this whole method as currently constructed kinda pretty much is bad.

Can we do the following?

- create a constant hash of image size values
```
IMAGE_SIZES = {
  small: xx,
  medium: xx,
  large: xx,
  etc: xx
}.with_indifferent_access.freeze
```
- find our size in that hash and apply some sensible default
```
gravatar_url(size: IMAGE_SIZES[size || :medium])
```
- make it more explicit that avatar_kind is an attribute, and kind is a parameter
```
if kind || self.avatar_kind == 'gravatar'
```